### ISSUE TITLE
[pidgin] Package failed to build

### ISSUE THREAD

**Issue by mayd** [msys2/MINGW-packages#8625]
I attempted to build  x86_64 and i686 packages for pidgin from the mingw repository source.

makepkg-mingw failed with the following error messages (after printing many warnings):

```
ERROR: 'IT_PROG_INTLTOOL' must appear in configure.ac for intltool to work.
autoreconf: error: intltoolize failed with exit status: 1
==> ERROR: A failure occurred in prepare().
    Aborting...
```

The full log file created by makepkg-mingw is attached.

[mingw-w64-pidgin-2.11.0-1-x86_64-prepare.log](https://github.com/msys2/MINGW-packages/files/6450310/mingw-w64-pidgin-2.11.0-1-x86_64-prepare.log)

**Comment by mayd** [comment]
After some searching on the Web I found the following curious lines in the (source code for intltool on Github)[https://github.com/cgwalters/intltool-git-mirror/blob/master/intltoolize.in]

157  if test -z "$automake"; then
158    if egrep '^(AC|IT)_PROG_INTLTOOL' $configure >/dev/null 2>&1; then :
159    else
160      echo "ERROR: 'IT_PROG_INTLTOOL' must appear in $configure for intltool to work."
161      exit 1
162    fi

It seems that the symbols IT_PROG_INTLTOOL and AC_PROG_INTLTOOL can only appear at the *beginning* of a line.
In the file configure.ac for the pidgin source code I note that the symbol AC_PROG_INTLTOOL is indented. 

Anyway, I added a small patch to PKGBUILD to fix the indentation, and also added the command `automake --add-missing` to the `prepare()` function because it seems to be recommended practice when updating autoconf-based builds.

Now pidgin packages build for both x86_64 and x86-i686 architectures.  And the installed packages appear to work as expected.
